docs/guides/connecting_to_your_data/filesystem/pandas.md

```diff
@@ -0,0 +1,136 @@
+
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas.py#L23
+```
+
+</TabItem>
+</Tabs>
+
+### 3. Write a `BatchRequest`
+
+In a `RuntimeBatchRequest` the `data_asset_name` can be any unique name to identify this batch of data. Please update the `data_asset_name` to something meaningful to you.
+
+<Tabs
+  defaultValue="csv"
+  values={[
+    {label: 'CSV', value: 'csv'},
+    {label: 'CSV with custom delimiters', value: 'csv_delimiters'},
+    {label: 'CSV without a header', value: 'csv_headerless'},
+  ]}>
+  <TabItem value="csv">
+
+Add the path to your csv on a filesystem to the `path` key under `runtime_parameters` and run the following code:
+
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas.py#L24-L31
+```
+
+  </TabItem>
+  <TabItem value="csv_delimiters">
+
+  **TODO** do this
+
+  </TabItem>
+  <TabItem value="csv_headerless">
+
+  **TODO** do this
+
+  </TabItem>
+</Tabs>
+
+### 4. Test your datasource configuration by getting a `Batch`
+
+Load a `Batch` of your data to test if your `Datasource` configuration works by running the following code:
+
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas.py#L39
+```
+
+Congratulations! If no errors are shown here, you've just connected to your data on a filesystem using pandas!
+
+:::tip Next steps
+After connecting to data, usually people validate data!
+**TODO** insert link or have this be an include.
+
+Right now a Batch is just Data + Metadata that is recognized by GE. If you want to actually Validate your data with Expectations, you will need a Batch + ExpectationSuite.
+:::
+
+## Additional Notes
+
+**TODO**
+Consider removing learning-focused material.
+
+:::note What does this configuration contain?
+
+- It has a `PandasExecutionEngine` which provides the computing resources that will be used to perform validation on your data.
+- It also has a default `RuntimeDataConnector` named: `default_runtime_data_connector_name` (**TODO** discuss) which will loads your data into a `Batch`.
+- It also has a default batch_identifier : `default_identifier_name` (**TODO** discuss) used to uniquely identify Batches
+:::
+
+:::note A word about DataConnectors
+A `RuntimeDataconnector` is usually enough to test whether your connection works (as we are doing now), or in cases where you are using Great Expectations to interactively validate a few Batches of Data.
+
+Great Expectations also has something called an `ActiveDataConnector` (Configured or Inferred) that can be used to automate the retrieval and validation of Batches while also enabling more sophisticated filtering and sorting. Please look at (**TODO** insert link) for more information.
+:::
+
+### Full script
+
+Here is the full script from which snippets were taken.
+
+```python file=../../../../integration/code/connecting_to_your_data/filesystem/pandas.py#L1-L40
+```
```

Comment:
I personally think including this would be helpful (maybe as a collapsable section that is not displayed by default?) and would like to hear what other pod members think.